docker-compose.yml

```diff
@@ -40,7 +36,7 @@ services:
     # are configured via the ServerAddress field of cmd.GRPCClientConfig.
     # TODO: Remove this when ServerAddress is deprecated in favor of SRV records
     # and DNSAuthority.
-    dns: 10.55.55.10
+    dns: 10.77.77.10
```

Comment:
I think this is the key bit for the consul part of this change, where we tell Boulder how to find bconsul at the non-55.55 address. Is that right?

In each of our JSON configs, we specify `"dnsAuthority": "consul.service.consul",`. Boulder then looks up that name using the system DNS (which is handled by Docker). Since Docker doesn't know that name, it queries the IP address configured here (`10.77.77.10`). So `10.77.77.10` (consul) is getting a query for `consul.service.consul`. Self-referential but it should work! One thing I don't understand: `test/consul/config.hcl` doesn't have an entry for a service named `consul`. Do you know where the answer for that is coming from?